beacon_node/network/src/sync/tests/range.rs

```diff
@@ -449,7 +449,15 @@ fn build_rpc_block(
             RpcBlock::new(None, block, Some(blobs.clone())).unwrap()
         }
         Some(DataSidecars::DataColumns(columns)) => {
-            RpcBlock::new_with_custody_columns(None, block, columns.clone(), spec).unwrap()
+            RpcBlock::new_with_custody_columns(
+                None,
+                block,
+                columns.clone(),
+                // TODO
+                columns.len(),
```

Comment:
Assumes the CGC value is equal to all the columns received, which makes sense. We can change it later if we want to do more complex tests

Reply:
Sounds good to me - in that case we could remove the TODO or add a note for future readers.